pkg/test/server.go

```diff
@@ -48,7 +55,10 @@ func (te *TestEnvironment) Start() (*rest.Config, error) {
 		if err == nil {
 			break
 		}
-		te.ControlPlane = integration.ControlPlane{}
+		te.ControlPlane = integration.ControlPlane{
```

Comment:
This doesn't look right. Don't we want to do this step before we issue `te.ControlPlane.Start()` ? Probably thats the reason `test.sh` travis test is passing which should infact fail for this PR because it is ignoring the environment variables for tests.